controllers/drcluster_controller.go

```diff
@@ -449,7 +449,7 @@ func validateCIDRsFormat(drcluster *ramen.DRCluster, log logr.Logger) error {
 	return nil
 }
 
-func (r DRClusterReconciler) processDeletion(u *drclusterInstance) (ctrl.Result, error) {
+func processDeletion(u *drclusterInstance) (ctrl.Result, error) {
```

Comment:
I prefer to keep this as a method. You can probably remove the receiver like you have in other cases.

Reply:
Done